docker/up.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e # exit on an error
+
+ERROR(){
+    /bin/echo -e "\e[101m\e[97m[ERROR]\e[49m\e[39m $@"
+}
+
+WARNING(){
+    /bin/echo -e "\e[101m\e[97m[WARNING]\e[49m\e[39m $@"
+}
+
+INFO(){
+    /bin/echo -e "\e[104m\e[97m[INFO]\e[49m\e[39m $@"
+}
+
+exists() {
+    type $1 > /dev/null 2>&1
+}
+
+
+exists ssh-keygen || { ERROR "Please install ssh-keygen (apt-get install openssh-client)"; exit 1; }
+exists docker || { ERROR "Please install docker (https://docs.docker.com/engine/installation/)"; exit 1; }
+exists docker-compose || { ERROR "Please install docker-compose (https://docs.docker.com/compose/install/)"; exit 1; }
+
+if [ ! -f ./secret/node.env ]; then
+    INFO "Generating key pairs"
+    ssh-keygen -t rsa -N "" -f ./secret/id_rsa
+    
+    INFO "Generating ./secret/control.env"
+    echo '# generated by jepsen/docker/up.sh, parsed by jepsen/docker/control/bashrc' > ./secret/control.env
+    echo '# NOTE: \\n is expressed as ↩' >> ./secret/control.env
+    echo SSH_PRIVATE_KEY="$(cat ./secret/id_rsa | perl -p -e 's/\n/↩/g')" >> ./secret/control.env
+    echo SSH_PUBLIC_KEY=$(cat ./secret/id_rsa.pub) >> ./secret/control.env
+    
+    INFO "Generating ./secret/node.env"
+    echo '# generated by jepsen/docker/up.sh, parsed by the "tutum/debian" docker image entrypoint script' > ./secret/node.env
+    echo ROOT_PASS=root >> ./secret/node.env
```

Comment:
I'm not sure why we need `ROOT_PASS` here.
Just keeping the original behavior.
ping @tjake